packages/lodestar/test/unit/chain/stateTransition/util/seed.test.ts

```diff
@@ -1,10 +1,10 @@
 import { assert } from "chai";
 
-import {config} from "../../../../../src/config/presets/mainnet";
+import {config} from "@chainsafe/eth2.0-config/lib/presets/mainnet";
```

Comment:
All of the eth2.0-configs, are accessing a layer too deep i think?

Reply:
This is how it "should" be, for some version of should.
We aren't exporting the presets to the index, so the expected usage pattern is to import from the file directly.